recipes/mikktspace/all/conanfile.py

```diff
@@ -0,0 +1,75 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+
+
+class MikkTSpaceConan(ConanFile):
+    name = "mikktspace"
+    description = " A common standard for tangent space used in baking tools to produce normal maps."
+    homepage = "https://github.com/mmikk/MikkTSpace"
+    url = "https://github.com/conan-io/conan-center-index"
+    license = "Zlib"
+    topics = ("conan", "tangent", "space", "normal")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False
+    }
+
+    generators = "cmake"
+    exports_sources = ['CMakeLists.txt', 'patches/*']
```

Comment:
```suggestion
    exports_sources = ['CMakeLists.txt']
```

There is no patch to be applied